deepspeed/runtime/fp16/onebit_adam.py

```diff
@@ -0,0 +1,374 @@
+'''
+Copyright 2019 The Microsoft DeepSpeed Team
```

Comment:
2019 -> 2020

Reply:
Done. I thought this will happen when we grep for all others that are 2019 as well :)